Cleaned up audit event fields.

edewata opened the pull request:
The audit-events.properties has been modified to list the
fields used by each event. The field descriptions will be
added separately later.

https://pagure.io/dogtagpki/issue/2686

geetikakay commented:
@edewata I think for CERT_SIGNING_INFO it will have SKI(mandatory) but it can have authority id as well for subca(optional) for audit logs.Also, I think for CRL_SIGNING_INFO it will not have authority id. 

For failure event, in FULL and DELTA CRL GENERATION, CRLNum is not there.

edewata commented:
@geetikakay Thanks for the review! I've added AuthorityID to CERT_SIGNING_INFO (I missed that) and removed it from CRL_SIGNING_INFO (the code is there, but it's not actually used). For FULL/DELTA_CRL_GENERATION I keep the CRLNum since the event description describes both the successful and failure cases, so the CRLNum description needs to explain when the field will or will not be available.
